.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -78,8 +79,10 @@ body:
   - type: textarea
     id: sdk-setup
     attributes:
-      label: SDK Setup
-      description: How do you set up your Sentry SDK? Please show us your `Sentry.init` options.
+      label: SDK Setup/Reproduction Example
+      description:
+        How do you set up your Sentry SDK? Please show us your `Sentry.init` options.
+        Or even better—share a link to a reproduction example.
```

Comment:
```suggestion
        Or even better — share a link to a reproduction example.
```

without a space this looks a bit weird in github IMHO 🤔

Reply:
But is is the correct way of writing 😄  ([source](https://www.thepunctuationguide.com/em-dash.html))